crates/sui-core/src/transaction_driver/effects_certifier.rs

```diff
@@ -188,6 +188,7 @@ impl EffectsCertifier {
         }
     }
 
+    #[instrument(level = "debug", skip_all, err, fields(tx_digest = ?tx_digest, consensus_position = ?consensus_position, ret_effects_digest = tracing::field::Empty))]
```

Comment:
For ret_effects_digest, why not keep it in a log message? Tbh, besides tx digest and maybe consensus position, I think we can just log these IDs once and not add them to the span, to reduce logging volume? Or do you find them useful in traces?

Reply:
To be fair this is mostly for completeness. I wouldn't really worry about logging volume. But during an investigation the effects digest will be nicely attached there .